### PR TITLE
[Call of Cthulhu 7th Ed] Update sv.json

### PR DESCRIPTION
**Pull request by DoktorLaser**
I have attempted to translate the character sheet according to the terminology used in the Swedish edition of Call of Cthulhu, released by Eloso Förlag AB in 2020. No concern has yet been given regarding how the translated terms affect the layout.

## Roll20 Requests

Comments are very helpful for reviewing the code changes. Please answer the relevant questions below in your comment.

- [ ] Does the pull request title have the sheet name(s)? Include each sheet name.
- [ ] Is this a bug fix?
- [ ] Does this add functional enhancements (new features or extending existing features) ?
- [ ] Does this add or change functional aesthetics (such as layout or color scheme) ? 
- [ ] If changing or removing attributes, what steps have you taken, if any, to preserve player data ?
- [ ] If this is a new sheet, did you follow [Building Character Sheets standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) ?

If you do not know English. Please leave a comment in your native language.

### CODE REVIEW

File: Call_of_Cthulhu_7th_Ed/translations/sv.json [hunk 1]
```diff
@@ -1,394 +1,394 @@
...
+	"Other Skill #8": "Annan färdighet #8",
+	"Other Skill #9": "Annan färdighet #9",
+	"Other Skill #10": "Annan färdighet #10",
+	"spells-u": "Besvärjelser",
+	"companions&mounts-u": "Kumpaner & riddjur",
+	"vehicles-u": "Fordon",
+	"cap-u": "Kapacitet",
+	"vehicleattack-u": "Kollisionsattack",
+	"operatorskill": "Förarens färdighet",
+	"MountedWeaponSkill": "Monterad vapenfärdighet",
```
Comment (LarsSjogreen) [inline Call_of_Cthulhu_7th_Ed/translations/sv.json:251]: This sounds a bit weird or archaic. Maybe "Beriden vapenfärdighet" or "Vapenfärdighet till häst" would work better?

File: Call_of_Cthulhu_7th_Ed/translations/sv.json [hunk 2]
```diff
@@ -1,394 +1,394 @@
...
-	"dam": "Dam:",
-	"pulp-bout-of-madness-u-time": "Pulp Bout of Madness - Real-Time",
-	"crazy-pan": "Crazy Plan:",
-	"rage": "Rage:",
-	"gloat": "Gloat:",
-	"relax": "Relax:",
-	"show-off": "Show Off:",
+	"paranoia-dec": "Utredaren drabbas av paranoia under 1D10 rundor: alla är ute efter henne, det går inte att lita på något, hon blir spionerad på, någon har förrått henne, vad hon ser är bluff.",
+	"significant-person": "Viktig anhörig:",
+	"significant-person-dec": "Leta efter Viktiga anhöriga i utredarens bakgrund. Utredaren misstar en närvarande individ för att vara en av dessa viktiga personer. Fundera över hur relationen fungerar, varefter utredaren agerar utifrån det under 1D10 rundor.",
```
Comment (LarsSjogreen) [inline Call_of_Cthulhu_7th_Ed/translations/sv.json:323]:
```suggestion
	"significant-person-dec": "Leta efter viktiga anhöriga i utredarens bakgrund. Utredaren misstar en närvarande individ för att vara en av dessa viktiga personer. Fundera över hur relationen fungerar, varefter utredaren agerar utifrån det under 1D10 rundor.",
```

File: Call_of_Cthulhu_7th_Ed/translations/sv.json [hunk 3]
```diff
@@ -1,394 +1,394 @@
...
+	"physical-hysterics-or-emotional-outburst": "Fysisk hysteri eller känslomässigt utbrott:",
+	"physical-hysterics-or-emotional-outburst-dec": "Utredaren är tillfälligt handlingsförlamad av skratt, gråt, skrikande, etc. under 1D10 rundor.",
+	"phobia": "Fobi:",
+	"phobia-dec": "Utredaren får en ny fobi. Slå 1D100 på Tabellen för Fobier eller så kan du välja en lämplig. Även om upphovet till fobin inte finns närvarande kommer utredaren att föreställa sig den under de följande 1D10 rundorna.",
+	"mania": "Mani:",
+	"mania-dec": "Utredaren får en ny mani. Slå 1D100 på Tabellen för Manier, eller så kan du välja en lämplig. Utredaren försöker hänge sig åt sin nya mani under de följande 1D10 rundorna.",
+	"amnesia-dec2": "Utredaren vaknar upp på en obekant plats utan minne om vem hon är. Hennes minnen kommer sakta tillbaka i sinom tid.",
+	"robbed": "Rånad:",
+	"robbed-dec": "Utredaren kommer till sans 1D10 timmar senare och upptäcker att hon blivit rånad (dock utan att ha blivit skadad). Om hon hade sig sin Värdefulla Ägodel får hon slå ett Turslag; misslyckas slaget är Ägodelen borta. Oavsett resultatet är alla andra föremål av intresse stulna.",
```
Comment (LarsSjogreen) [inline Call_of_Cthulhu_7th_Ed/translations/sv.json:336]:
```suggestion
	"robbed-dec": "Utredaren kommer till sans 1D10 timmar senare och upptäcker att hon blivit rånad (dock utan att ha blivit skadad). Om hon hade på sig sin Värdefulla Ägodel får hon slå ett Turslag; misslyckas slaget är Ägodelen borta. Oavsett resultatet är alla andra föremål av intresse stulna.",
```

File: Call_of_Cthulhu_7th_Ed/translations/sv.json [hunk 4]
```diff
@@ -1,394 +1,394 @@
...
+	"battered": "Mörbultad:",
+	"battered-dec": "Utredaren kommer till sans 1D10 timmar senare, mörbultad och i dåligt skick. Hon har förlorat hälften av de Kroppspoäng hon hade innan hon blev vansinnig (detta leder dock inte till allvarlig skada).",
+	"violence-dec2": "Utredaren förlorar alla hämningar och går lös på sin omgivning. När utredaren vaknar till sans behöver det inte vara omedelbart uppenbart vad eller vem hon har skadat. Det kan vara så att utredaren rent av dödat någon, men detta är upp till spelledaren.",
+	"ideology-beliefs": "Övertygelser:",
+	"ideology-beliefs-dec": "Utredaren tar en av sina Övertygel- ser från sin bakgrund och utagerar den på ett galet och demonstrativt sätt. En religiös person kan till exempel hålla en hätsk predikan på spårvagnen och anklaga folk för att vara syndare.",
```
Comment (LarsSjogreen) [inline Call_of_Cthulhu_7th_Ed/translations/sv.json:341]:
```suggestion
	"ideology-beliefs-dec": "Utredaren tar en av sina Övertygelser från sin bakgrund och utagerar den på ett galet och demonstrativt sätt. En religiös person kan till exempel hålla en hätsk predikan på spårvagnen och anklaga folk för att vara syndare.",
```

File: Call_of_Cthulhu_7th_Ed/translations/sv.json [hunk 5]
```diff
@@ -1,394 +1,394 @@
...
+	"pulp-mania-dec": "Hjälten erhåller en ny mani. Slå 1D100 på Tabellen för Manier, eller så kan du välja en lämplig. Utredaren försöker hänge sig åt sin nya mani.",
+	"pulp-bout-of-madness-u-summary": "Pulpvansinnesepisod - Sammanfattning",
+	"institutionalized-or-apprehended": "Intagen eller Gripen:",
+	"pulp-amnesia-dec2": "Hjälten kommer till sans på en obekant plats, utan minne om vem hon är och utan sina pulptalanger. Hjältens minne återvänder sakta med tiden. Pulptalangerna återvänder endast under en krissituation. I det här fallet innebär en krissituation då någons liv står på spel. När någons liv är i fara, slår hjälten ett Turslag; om det lyckas, så återvänder talangerna. Misslyckas slaget kan hjälten försöka igen efter 1D10 rundor.",
+	"pulp-robbed-dec": "Hjälten kommer till sans 1D10 timmar senare och upptäcker att hon blivit rånad (dock utan att ha blivit skadad). Om hon hade sig sin Värdefulla Ägodel får hon slå ett Turslag; misslyckas slaget är Ägodelen borta. Oavsett resultatet är alla andra föremål av intresse stulna.",
```
Comment (LarsSjogreen) [inline Call_of_Cthulhu_7th_Ed/translations/sv.json:379]:
```suggestion
	"pulp-robbed-dec": "Hjälten kommer till sans 1D10 timmar senare och upptäcker att hon blivit rånad (dock utan att ha blivit skadad). Om hon hade på sig sin Värdefulla Ägodel får hon slå ett Turslag; misslyckas slaget är Ägodelen borta. Oavsett resultatet är alla andra föremål av intresse stulna.",
```

File: Call_of_Cthulhu_7th_Ed/translations/sv.json [hunk 6]
```diff
@@ -1,394 +1,394 @@
...
+	"pulp-significant-people-dec": "Hjälten försöker under 1D10 timmar få tag i en av sina Viktiga anhöriga från sin bakgrund.",
+	"pulp-institutionalized-or-apprehended-dec": "Hjälten vaknar upp i en sjukhussäng eller i en fängelsecell. Hennes minnen om varför hon hamnat där kan sakta men säkert börja komma tillbaka.",
+	"pulp-flee-in-panic-dec2": "När hjälten blir sig själv igen vet hon inte längre var hon befinner sig. Hon kan vara vilse ute i skogen, irra runt på okända bakgator eller redan sitta på tåget med nyinköpt biljett i hand.",
+	"pulp-phobia-dec2": "Hjälten får en ny fobi. Slå 1D100 på Tabellen för Fobier eller så kan du välja en lämplig. Hjälten kommer till sans 1D10 timmar senare efter att ha försökt att undvika sin nya fobi till det yttersta.",
+	"pulp-mania-dec2": "Hjälten får en ny mani. Tabellen för Manier, eller så kan du välja en lämplig. Under 1D10 timmar hänför sig hjälten till sin nya mani fullt ut.",
```
Comment (LarsSjogreen) [inline Call_of_Cthulhu_7th_Ed/translations/sv.json:387]:
```suggestion
	"pulp-mania-dec2": "Hjälten får en ny mani. Slå 1D100 på Tabellen för Manier, eller så kan du välja en lämplig. Under 1D10 timmar hänför sig hjälten till sin nya mani fullt ut.",
```

File: Call_of_Cthulhu_7th_Ed/translations/sv.json [hunk 7]
```diff
@@ -1,394 +1,394 @@
...
+	"delusion-da-dec": "Utredaren ser saker som inte finns, men är oförmögen att skilja verklighet från vanföreställning.",
+	"delusion-da": "Vanföreställning:",
+	"shackled-da": "Fängslad:",
+	"delusion-da-dec2": "Utredaren har påverkats märkbart av en hallucination, vilket fått henne att handla på något sätt. Kanske genom att gräva en djup grop att gömma sig i, eller förskansat sig själv innuti en hydda.",
```
Comment (LarsSjogreen) [inline Call_of_Cthulhu_7th_Ed/translations/sv.json:391]:
```suggestion
	"delusion-da-dec2": "Utredaren har påverkats märkbart av en hallucination, vilken fått henne att handla på något sätt. Kanske genom att gräva en djup grop att gömma sig i, eller förskansat sig själv inuti en hydda.",
```

File: Call_of_Cthulhu_7th_Ed/translations/sv.json [hunk 8]
```diff
@@ -1,394 +1,394 @@
...
+	"flee-in-panic-da-dec2": "När utredaren kommer till sans befinner hon sig långt borta, kanske vilse i vildmarken eller bankande på en dörr med krav om att släppas in.",
+	"shackled-da-dec": "Utredaren, som yrar och talar tungomål, har spärrats in och försatts med försatts med bojor eller fängslats på något vis."
```
Comment (LarsSjogreen) [inline Call_of_Cthulhu_7th_Ed/translations/sv.json:393]:
```suggestion
	"shackled-da-dec": "Utredaren, som yrar och talar tungomål, har spärrats in och försatts med bojor eller fängslats på något vis."
```

**Review by LarsSjogreen — CHANGES_REQUESTED** [review]
Some small typo's here and there.

File: Call_of_Cthulhu_7th_Ed/translations/sv.json (hunk 1 above)
Comment (DoktorLaser) [reply Call_of_Cthulhu_7th_Ed/translations/sv.json:251]: Thaaank you! Of course it's "Beriden vapenfärdighet". I was completely stuck thinking in terms of vehicles, looking for terminology regarding weapons mounted on them, that I didn't even stop to think about mounted as in riding...

Comment (DoktorLaser) [inline Call_of_Cthulhu_7th_Ed/translations/sv.json:251]:
```suggestion
	"MountedWeaponSkill": "Beriden vapenfärdighet",
```

File: Call_of_Cthulhu_7th_Ed/translations/sv.json [hunk 9]
```diff
@@ -1,394 +1,394 @@
...
+	"significant-people": "Viktig anhörig:",
+	"significant-people-dec": "Utredaren försöker under 1D10 timmar få tag i en av sina Viktiga anhöriga från sin bakgrund.",
+	"institutionalized": "Intagen:",
+	"institutionalized-dec": "Utredaren vaknar upp i en sjukhussäng eller i en fängelsecell. Hennes minnen om varför hon hamnat där kan sakta men säkert börja komma tillbaka.",
+	"flee-in-panic-dec2": "När utredaren blir sig själv igen vet hon inte längre var hon befinner sig. Hon kan vara vilse ute i skogen, irra runt på okända bakgator eller redan sitta på tåget med nyinköpt biljett i hand.",
+	"phobia-dec2": "Utredaren får en ny fobi. Slå 1D100 på Tabellen för Fobier eller så kan du välja en lämplig. Utredaren kommer till sans 1D10 timmar senare efter att ha försökt att undvika sin nya fobi till det yttersta.",
+	"mania-dec2": "Utredaren får en ny mani. Tabellen för Manier, eller så kan du välja en lämplig. Under 1D10 timmar hänför sig utredaren till sin nya mani fullt ut.",
```
Comment (DoktorLaser) [inline Call_of_Cthulhu_7th_Ed/translations/sv.json:348]:
```suggestion
	"mania-dec2": "Utredaren får en ny mani. Slå 1D100 på Tabellen för Manier, eller så kan du välja en lämplig. Under 1D10 timmar hänför sig utredaren till sin nya mani fullt ut.",
```

File: Call_of_Cthulhu_7th_Ed/translations/sv.json (hunk 1 above)
Comment (LarsSjogreen) [reply Call_of_Cthulhu_7th_Ed/translations/sv.json]: 👍

**Review by LarsSjogreen — APPROVED** [review]
Looks good to me.

**Comment by jomblr** [comment]
Looks great!

**Comment by mperes** [comment]
Hello @DoktorLaser ! Your Pull Request includes changes to translation files. We will not deny your Pull Request, but we would prefer sheet contributors use Crowdin for translations. Please email Cassie at cassie@roll20.net and she will get you an invite to our Crowdin. Be sure to include the languages you would like to translate, and please use Crowdin for future translation updates! Thanks for all you do!

**Pull request merged**